rio_cogeo/scripts/cli.py

```diff
@@ -84,9 +85,9 @@ def cogeo(
 
     config = dict(
         NUM_THREADS=threads,
-        BIGTIFF="IF_SAFER",
-        GDAL_TIFF_INTERNAL_MASK=True,
-        GDAL_TIFF_OVR_BLOCKSIZE=block_size,
+        BIGTIFF=os.environ.get("BIGTIFF", "IF_SAFER"),
+        GDAL_TIFF_INTERNAL_MASK=os.environ.get("GDAL_TIFF_INTERNAL_MASK", True),
+        GDAL_TIFF_OVR_BLOCKSIZE=os.environ.get("GDAL_TIFF_OVR_BLOCKSIZE", block_size),
```

Comment:
@vincentsarago is the intent here to let a user override `GDAL_TIFF_INTERNAL_MASK=True` by providing a false value?

Reply:
yes